src/resource/solidDataset.ts

```diff
@@ -1133,3 +1137,37 @@ function resolveLocalIrisInThing(
       : thing.url,
   });
 }
+
+/**
+ * Fetch the contents of '.well-known/solid' for a given resource URL.
+ *
+ * @param url URL of a Resource.
+ * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).
+ * @returns Promise resolving to a [[SolidDataset]] containing the data at '.well-known/solid' for the given Resource, or rejecting if fetching it failed.
+ */
+export async function getWellKnownSolid(
+  url: UrlString | Url,
+  options: Partial<
+    typeof internal_defaultFetchOptions & ParseOptions
+  > = internal_defaultFetchOptions
+): Promise<SolidDataset & WithServerResourceInfo> {
+  const urlString = internal_toIriString(url);
+  const resourceMetadata = await getResourceInfo(urlString, options);
+  const linkedResources = getLinkedResourceUrlAll(resourceMetadata);
+  const rootResources =
+    linkedResources["http://www.w3.org/ns/pim/space#storage"];
```

Comment:
IRIs which are part of the spec are typically added to the `src/constants.ts` file. At some point, we should consider using the Artifact Generator for this :)